ring-core/src/ring/middleware/resource.clj

```diff
@@ -3,7 +3,8 @@
   (:require [ring.util.codec :as codec]
             [ring.util.response :as response]
             [ring.util.request :as request]
-            [ring.middleware.head :as head]))
+            [ring.middleware.head :as head]
+            [clojure.string :as s]))
```

Comment:
Sorry to be picky, but could this be aliased to `str` instead of `s`, both for clarity and consistency in the codebase.